package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "coffee-shop",
   "private": true,
-  "dependencies": {},
+  "dependencies": {
```

Comment:
WHy do you wanna change this config ?

Reply:
i changed nothing, i just did `npm install husky` and fixed runs with brew doctor & brew update

Reply:
@Crysicia ok checked the conflicts below, its because i installed husky without the --save-dev option ...